pynestml/codegeneration/resources_nest/point_neuron/common/NeuronClass.jinja2

```diff
@@ -116,6 +118,26 @@ template <> void RecordablesMap<{{neuronName}}>::create()
   }
 }
 
+{%- if neuron.get_spike_input_ports()|length >1 %}
+std::map<std::tuple< int, std::string >, int> {{neuronName}}::rport_to_nestml_port =
+{
+    {%- for key, values in utils.get_spike_input_ports_in_pairs(neuron).items() %}
```

Comment:
Would you mind changing the indentation here so that the ``{%`` is always at the start of the line, followed by as much indentation is necessary for the jinja bits? E.g.
```Jinja
{% for foo in bar %}
{%   if baz == "xyzzy" %}
{%     set doge = 42 %}
```